### ISSUE TITLE
Helm "enable metrics" boolean

### ISSUE THREAD

**Issue by jcpunk** [projectcalico/calico#6887]
## Expected Behavior
A number of common helm charts have a value you can set to enable metrics reporting.  Since calico has a metrics interface, having a clean way to enable it via helm would be nice.

## Current Behavior
You must manually patch the instance to enable metrics

## Possible Solution
Add a boolean to `values.yml` and the relevant templates to enable the metrics service on request.

## Steps to Reproduce (for bugs)

1. Install via helm
2. look for way to enable metrics as a parameter
3.
4.

## Context

A consistent way to enable metrics across a wide array of helm charts makes things easier.

## Your Environment

* Calico version: 3.24.2
* Orchestrator version (e.g. kubernetes, mesos, rkt): 1.25.2
* Operating System and version: CentOS Stream9
* Link to your project (optional):

**Comment by jcpunk** [comment]
Neat, I didn't see those in there.  Can it be added to the documentation?

**Comment by caseydavenport** [comment]
Yeah, the documentation isn't very clear at the moment. We need to do better.

The `installation` section contains all of the fields specified in the InstallationSpec here: https://projectcalico.docs.tigera.io/reference/installation/api#operator.tigera.io/v1.InstallationSpec

Maybe there's a way we can auto-generate the values.yaml doc to include all of those fields, but for now that link is effectively the documentation.

**Comment by mgleung** [comment]
@jcpunk out of curiosity, which pages of the documentation do you normally look at for this? We have the fields in https://projectcalico.docs.tigera.io/reference/installation/api but it seems that it's not a great UX. Any feedback on how we can better arrange this information is welcome.

**Comment by mgleung** [comment]
OK, sounds like the right path forward is to beef up the documentation in the README for now. If there are any other suggestions you have on how to make this easier to read going forward, please let us know!

**Comment by lou-lan** [comment]
> OK, sounds like the right path forward is to beef up the documentation in the README for now. If there are any other suggestions you have on how to make this easier to read going forward, please let us know!

@mgleung   @caseydavenport  Automatically generating the README.md document for the Chart could make the project easier to deploy. If there are enhancement plans, I think I could do this work and the following are the planned steps.

* Select a tool for automatically generating the README.md document.
  * https://github.com/norwoodj/helm-docs
  * https://github.com/bitnami-labs/readme-generator-for-helm
* Add comments to the values.yaml.
* Add a step to the Makefile, for example `make helm-chart-docs`.
